doc/build_ref/TrilinosBuildReferenceTemplate.rst

```diff
@@ -105,3 +105,65 @@ In order do co-development of TriBTS and Trilinos (see http://http://trac.trilin
+  | Build with separate libs   | KOKKOS_SEPARATE_LIBS                         |
+  +----------------------------+----------------------------------------------+
+  | Debug builds               | KOKKOS_DEBUG                                 |
+  +----------------------------+----------------------------------------------+
+  | Devices Options                                                           |
+  +----------------------------+----------------------------------------------+
+  | Enable Cuda                | TPL_ENABLE_CUDA                              |
+  +----------------------------+----------------------------------------------+
+  | Enable OpenMP              | Trilinos_ENABLE_OpenMP                       |
+  +----------------------------+----------------------------------------------+
+  | Enable Pthread             | TPL_ENABLE_PThread                           |
+  +----------------------------+----------------------------------------------+
+  | Specify Serial             | TPL_ENABLE_MPI (not)                         |
+  +----------------------------+----------------------------------------------+
+  | Advanced options                                                          |
+  +----------------------------+----------------------------------------------+
+  | Enable compiler warnings   | KOKKOS_ENABLE_COMPILER_WARNINGS              |
+  +----------------------------+----------------------------------------------+
+  | Aggressive Vectorization   | KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION       |
+  +----------------------------+----------------------------------------------+
+  | Profiling                  | KOKKOS_ENABLE_PROFILING                      |
+  +----------------------------+----------------------------------------------+
+  | Enable profile load print  | KOKKOS_ENABLE_PROFILE_LOAD_PRINT             |
+  +----------------------------+----------------------------------------------+
+  | Enable dualview modify chk | KOKKOS_ENABLE_DUALVIEW_MODIFY_CHECK          |
+  +----------------------------+----------------------------------------------+
+  | TPLs                                                                      |
+  +----------------------------+----------------------------------------------+
+  | Use hwloc library          | TPL_ENABLE_HWLOC                             |
+  +----------------------------+----------------------------------------------+
+  | Use memkind library        | KOKKOS_ENABLE_MEMKIND                        |
+  +----------------------------+----------------------------------------------+
+  | Use librt                  | KOKKOS_ENABLE_LIBRT                          |
+  +----------------------------+----------------------------------------------+
+  | CUDA Options                                                              |
+  +----------------------------+----------------------------------------------+
+  | Enable CUDA LDG            | KOKKOS_ENABLE_CUDA_LDG_INTRINSIC             |
```

Comment:
Is the point of this list to document the available options, or just to list them?  I ask because you explain the UVM acronym below, but not the LDG acronym above.

Reply:
I am copying documentation from elsewhere in Kokkos so I was giving as much info as I have. 

I'll add (global memory load)